experimental/examples/bubblebench/tree/src/cursor/bubble.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IBubble } from "@fluid-example/bubblebench-common";
+import { Anchor, brand, FieldKey, ISharedTree } from "@fluid-internal/tree";
+import { SharedTreeNodeHelper } from "./tree-utils";
+
+export class Bubble implements IBubble {
+	static xFieldKey: FieldKey = brand("x");
```

Comment:
the main point of using branded strings for field names is to get use sites and declaration sites to use the same thing so they can't diverge. Redeclaring these here separate from the schema mostly defeats the point. If we are going to want statics that refer to these fields, we should prabably put them in a central palce which both this kind of code and the schema can use (or acccess them off the schema objects, thouhg I'm not sure we have a good API for that right now)